lib/albacore/task_types/test_runner.rb

```diff
@@ -191,7 +191,7 @@ def handle_directory dll, exe, &block
                 [Paths.normalise_slashes(Pathname.new(File.dirname(dll))), Pathname.new(exe)]
             end
 
-          exe_rel = exe.relative_path_from dir
+          exe_rel = exe.to_s.include?(':') ? exe : exe.relative_path_from(dir)
```

Comment:
Yes, but expand_path assumes that path separators are the same and will return false negatives cross-platform.